fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProviderSpec.groovy

```diff
@@ -206,6 +208,34 @@ class LuceneSearchProviderSpec extends SearchProviderSpec<LuceneSearchProvider>
         !Files.exists(file4)
     }
 
+    @Timeout(5)
+    def "If time waiting for write lock exceeds timeout fail query"() {
+        setup:
+        searchProvider.@searchTimeout = 2000
```

Comment:
Let's make this faster, like 500 milliseconds. Our tests take long enough as it is without adding one or more tests that are guaranteed to take 2 seconds.